### ISSUE TITLE
Filters doesn't work after pjax reload

### ISSUE THREAD

**Issue by marcuzy** [kartik-v/yii2-grid#487]
Hi,

I noticed that when a grid have been reloaded with pajax, then filters doesn't work (and checkboxes https://github.com/kartik-v/yii2-grid/issues/477). But  the pagination works fine. 

I found the trouble after updating to yii 2.0.8 (it works fine with 2.0.6)

```php
<?php echo GridView::widget([
        'dataProvider' => $dataProvider,
        'filterModel' => $searchModel,
        'id'=>'orders',
        'pjax' => true,
        'filterSelector' => '#per-page',
        'columns' => [
            //...
        ],
    ]); ?>
```
Thanks!

**Comment by almirb** [comment]
I'm experiencing the same issue using MaskedInput in gridview filters.  The mask disappear after a pjax reload.

**Comment by almirb** [comment]
I solved It with:
````js
$.pjax.defaults.scrollTo = false;
````

Also tried to put this setting internally into my Kartik gridview with the code below, to avoid global changes i in my pjax configuration:
````php
'pjaxSettings' => ['options' => ['scrollTo' => false]],
````
But It didn't work.

**Comment by kartik-v** [comment]
Developers need to reinitialize jquery plugins themselves after pjax reload. For Krajee widget based filters, this extension does this automatically.

**Comment by baldiri** [comment]
Hi,

i have same issue as well. 
I'm not sure about what kartik-v is saying with "Developers need to reinitialize jquery plugins themselves after pjax reload". How i must/can reinitialize jquery plugins after pjax reload?

Thanks in advance!

**Comment by necrox87** [comment]
I think this is the solution: https://lorenzo.mile.si/reinitialize-kartik-plugins-after-pjax-with-pjaxcontainerid/383/
